_posts/2023-11-02-election-competitiveness-in-charlotte-north-carolina/election-competitiveness-in-charlotte-north-carolina.Rmd

```diff
@@ -0,0 +1,76 @@
+---
+title: "Election Competitiveness in Charlotte, North Carolina"
+description: |
+  Analyzing changes in competitiveness for mayoral and city council elections from 2015 to 2022 in Charlotte, North Carolina.
+author: Angeline Zhao
+date: 2023-11-02
+output:
+  distill::distill_article:
+    self_contained: false
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+## City Council in Charlotte
```

Comment:
Maybe Charlotte, NC City Council is more clear. Currently you don't mention NC until the second section.